docs/source/index.rst

```diff
@@ -0,0 +1,104 @@
+.. _DNP3-Driver:
+
+===========
+DNP3 Driver
+===========
+
+VOLTTRON's DNP3 driver enables the use of `DNP3 <https://en.wikipedia.org/wiki/DNP3>`_ (Distributed Network Protocol)
+communications, reading and writing points via a DNP3 Outstation.
+
+Requirements
+============
+
+The DNP3 driver requires the `dnp3-python <https://github.com/VOLTTRON/dnp3-python>`_ package, a DNP3 Python
+implementation wrapping on the `opendnp3<https://github.com/dnp3/opendnp3>`_ package.
```

Comment:
Change to: `opendnp3 <https://github.com/dnp3/opendnp3>`_ (the missing space before the angle bracket makes the link render incorrectly).